# Initialize Tracker
Header header
CompressedImage img
string label
float32 roi[]
